pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerGauge.java

```diff
@@ -40,6 +40,9 @@ public enum ServerGauge implements AbstractMetrics.Gauge {
   REALTIME_SEGMENT_NUM_PARTITIONS("realtimeSegmentNumPartitions", false),
   LLC_SIMULTANEOUS_SEGMENT_BUILDS("llcSimultaneousSegmentBuilds", true),
   RESIZE_TIME_MS("milliseconds", false),
+  // metric tracking the freshness of real time ingested data
+  REALTIME_DATA_FRESHNESS_MS("milliseconds", false),
```

Comment:
Nit: suggest renaming the metric to something more descriptive like `PARTITION_INGESTION_LAG_MS`.

Reply:
I think we should also change `milliseconds` to something like `partition_ingestion_lag_ms`